tests/src/function.spec.ts

```diff
@@ -13,6 +13,34 @@ export = () => {
 		expect(add(123, 456)).to.equal(579);
 	});
 
+	it("should support no return value", () => {
+		function doStuff(doIHaveToMom: boolean) {
+			if (doIHaveToMom) {
+				return "cleans room";
+			} else {
+				return;
+			}
+		}
+
+		expect(doStuff(true)).to.equal("cleans room");
+		expect(doStuff(false)).never.to.be.ok();
+	});
+
+	/* https://github.com/roblox-ts/roblox-ts/issues/114
```

Comment:
This test is actually okay. The broken syntax was having the variables created before destructuring assignment.

i.e.
```TS
let a, b;
[a, b] = foo();
```